File: umbrella-insurance-backend/src/main/java/com/umbrella/insurance/endpoints/rest/users/v1/requests/UpdateUserRequest.java
package com.umbrella.insurance.endpoints.rest.users.v1.requests;

import lombok.Data;

@Data
public class UpdateUserRequest {
    private String username;
    private String emailAddress;
    private String phoneNumber;
    private String firstName;
    private String lastName;
    private Boolean consentedToEmails;
    private Boolean consentedToTexts;
    private String twoFactorAuthType;
    private String dateOfBirth;
}

File: umbrella-insurance-react-ui/src/endpoints/soa/user/v1/requests/UpdateUserRequest.ts
export class UpdateUserRequest {
    username?: string;
    emailAddress?: string;
    phoneNumber?: string;
    firstName?: string;
    lastName?: string;
    consentedToEmails?: boolean;
    consentedToTexts?: boolean;
    twoFactorAuthType?: string;
    dateOfBirth?: string;
}